sql/status_effects.sql

```diff
@@ -210,7 +210,7 @@ INSERT INTO `status_effects` VALUES (172,'intension',40,0,0,0,0,0,0,0,0);
 INSERT INTO `status_effects` VALUES (173,'dread_spikes',41,34,0,0,0,0,0,0,800);
 INSERT INTO `status_effects` VALUES (174,'magic_acc_down',16418,0,0,0,0,0,1,0,0);
 INSERT INTO `status_effects` VALUES (175,'magic_atk_down',16418,0,0,0,0,0,4,0,0);
-INSERT INTO `status_effects` VALUES (176,'quickening',4194476,0,0,0,0,0,0,0,0);
+INSERT INTO `status_effects` VALUES (176,'quickening',4194348,0,0,0,0,0,0,0,0);
```

Comment:
somebody meant to have job change in here, but they missed a zero before converting to decimal and wound up with synth support instead

changing this from
`40002C` (`4194348`)
to
`400002C` (`67108908`)

Reply:
also if its not supposed to be copy-able by drg's empathy 2c at the end becomes 25.

..and if generic damage when you weren't the target can take if then `EFFECTFLAG_DAMAGE = 0x00000010` needs added

I can test neither of these right now. but I'll accept the missing zero as "good enough" for this pr.

Reply:
I'm slow at hex math. But yes it should wear on damage, which I thought it did... I will revisit later tonight thanks!

Reply:
EFFECTFLAG_ON_JOBCHANGE = 0x00400000
This would mean theres no need to change that concrete one.

https://github.com/LandSandBoat/server/blob/base/src/map/status_effect.h#L68C1-L68C84

As for the other 2 cases, we can leave for now until we go test them

Reply:
/shrug

windows calculator, hex mode, add flags, then switch decimal. copy paste.